contrib/cube/meson.build

```diff
@@ -11,13 +11,15 @@ cube_scan = custom_target('cubescan',
 )
 generated_sources += cube_scan
 cube_sources += cube_scan
+bc_cube_sources = cube_sources
```

Comment:
Why is this variable needed at all? Why not just use `cube_sources` directly?

Reply:
`cube_sources` include the .h generated from bison as well.

Reply:
And why shouldn't those be in `bc_cube_sources`?

Reply:
If I add the full list it gonna end calling the generator for a .h file, example:
```
clang --emit-llvm [flags...] -o cube.bc cube.h
```
We only generate bitcode for .c not .h, the above is an error I found when trying to add directly the original cube_sources and the many other generated sources, we must filter out .h